custom-graders/README.md

```diff
@@ -46,23 +53,18 @@ Step 0 : FROM ubuntu:latest
 Successfully built 15c3a282b939
 ```
 
-### Test your docker grader locally with a sample submission.
-```sh
-$ export CustomGraderPath=$BASE_PATH/programming-assignments-demo/custom-graders
-$ docker run --user 1000 --net none -v $CustomGraderPath/FactoringGrader/SampleSubmission/:/shared/submission -t factoring_grader.v1.1
-```
+### Test graders locally using 'courseraprogramming'
+Testing in Coursera's production environment is quite different from testing the graders locally. 'courseraprogramming' is a software development toolkit that helps to develop asynchronous graders for Coursera (typically programming assignments). It can be used to test the graders locally in an environment that is somewhat similar to Coursera's production environment. Follow instructions [here](https://github.com/coursera/courseraprogramming) to get started.
 
-Output:
+After setting up courseraprogramming:
 ```sh
-{"isCorrect": true,"feedback": "Congrats! All test cases passed!"}
+$ export CustomGraderPath=$BASE_PATH/programming-assignments-demo/custom-graders/
+$ courseraprogramming grade local factoring_grader.v1.1 $CustomGraderPath/DemoAssignmentGrader/SampleSubmission/FactoringSampleSubmission partId HxbKF
 ```
 
-### Test graders locally using 'courseraprogramming'
-Testing in Coursera's production environment is quite different from testing the graders locally. 'courseraprogramming' is a software development toolkit that helps to develop asynchronous graders for Coursera (typically programming assignments). It can be used to test the graders locally in an environment that is somewhat similar to Coursera's production environment. Follow instructions [here](https://github.com/coursera/courseraprogramming) to get started.
-
 ### Package the docker image into a tar.
 ```sh
-$ docker save factoring_grader.v1.1 > factoring_grader.v1.1.tar
+$ docker save demo_grader.v1.1 > demo_grader.v1.1.tar
```

Comment:
Could you instead modify this to just use `courseraprogramming` instead? Thanks!